src/index.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const utilities = require("./utilities");
+const writeReadme = require("./readme");
+const { questions, installQuestions, testQuestions } = require("./questions");
+
+const start = async () => {
+  const readmeAnswers = await inquirer.prompt(questions);
+  const instructionsArray = [];
+  const testArray = [];
+
+  if (readmeAnswers.installationIncluded) {
+    let active = true;
+    while (active) {
+      const installationInstructions = await inquirer.prompt(installQuestions);
+      instructionsArray.push(installationInstructions.step);
+      if (!installationInstructions.continue) {
+        active = false;
+      }
+    }
+  }
+
+  if (readmeAnswers.testIncluded) {
+    let active = true;
+    while (active) {
+      const testInstructions = await inquirer.prompt(testQuestions);
+      testArray.push(testInstructions.step);
+      if (!testInstructions.continue) {
+        active = false;
+      }
+    }
+  }
+
+  const readmeDoc = writeReadme({
+    readmeAnswers,
```

Comment:
This would just need to be updated to reflect what you call the `rest` of the object in the initial destructuring on line 7.